tests/openedx_tagging/core/tagging/import_export/test_import_plan.py

```diff
@@ -299,11 +299,11 @@ def test_generate_actions(self, tags, replace, expected_errors, expected_actions
             "--------------------------------\n"
             "#1: Create a new tag with values (external_id=tag_31, value=Tag 31, parent_id=None).\n"
             "#2: Create a new tag with values (external_id=tag_32, value=Tag 32, parent_id=tag_1).\n"
-            "#3: Rename tag value of tag (external_id=tag_2) from 'Tag 2' to 'Tag 2 v2'\n"
-            "#4: Update the parent of tag (external_id=tag_4) from parent (external_id=tag_3) "
-            "to parent (external_id=tag_1).\n"
-            "#5: Rename tag value of tag (external_id=tag_4) from 'Tag 4' to 'Tag 4 v2'\n"
-            "#6: No changes needed for tag (external_id=tag_1)\n"
+            "#3: Rename tag value of <Tag> (tag_2 / Tag 2) to 'Tag 2 v2'\n"
+            "#4: Update the parent of <Tag> (tag_4 / Tag 4) from parent "
+            "<Tag> (tag_3 / Tag 3) to tag_1\n"
+            "#5: Rename tag value of <Tag> (tag_4 / Tag 4) to 'Tag 4 v2'\n"
+            "#6: No changes needed for <TagItem> (tag_1 / Tag 1)\n"
```

Comment:
From a user perspective, it's confusing that this sometimes says `<TagItem>` and other times says `<Tag>`. We may need to tweak this in the future, especially if we're trying to display this nicely in the frontend. (Though maybe we also need a JSON version of the plan for that.) I think this is OK for now though.